docker-app/qfieldcloud/core/admin.py

```diff
@@ -997,12 +1018,9 @@ def organization_owner__link(self, instance):
             instance.organization_owner, instance.organization_owner.username
         )
 
-    @admin.display(description=_("Storage"))
+    @admin.display(description=_("Total Storage"))
     def storage_usage__field(self, instance) -> str:
-        used_storage = filesizeformat10(instance.useraccount.storage_used_bytes)
-        free_storage = filesizeformat10(instance.useraccount.storage_free_bytes)
-        used_storage_perc = instance.useraccount.storage_used_ratio * 100
-        return f"{used_storage} {free_storage} ({used_storage_perc:.2f}%)"
+        return f"{instance.useraccount.current_subscription.plan.storage_mb} Mb"
```

Comment:
Is this causing a performance issue? The current calculation is quite handy for support.

Note that `instance.useraccount.current_subscription.plan.storage_mb` is not the same as `used_storage`.

I guess `storage_free_bytes` is having some performance issues, right?

Also, please use `filesizeformat10` for formatting, I guess it should be super cheap call.

Reply:
Yes the `storage_free_bytes` is expensive. 

Ok, will have a close look again.

Reply:
ok, so we agreed in chat we revert this change for this time and have a follow-up PR implementing this.